controllers/storageclassclaim_migration_controller.go

```diff
@@ -82,7 +82,7 @@ func (r *StorageClassClaimMigrationReconciler) Reconcile(ctx context.Context, re
 
 	storageClaim.Spec.EncryptionMethod = storageClassClaim.Spec.EncryptionMethod
 	storageClaim.Spec.StorageProfile = storageClassClaim.Spec.StorageProfile
-	storageClaim.Spec.StorageClient = storageClassClaim.Spec.StorageClient.DeepCopy()
+	storageClaim.Spec.StorageClient = storageClassClaim.Spec.StorageClient.DeepCopy().Name
```

Comment:
DeepCopy to get a string field? is that necessary?

Reply:
done.